services/tables/src/main/java/com/linkedin/openhouse/tables/config/TablesMvcConfigurer.java

```diff
@@ -42,7 +42,8 @@ public void addInterceptors(InterceptorRegistry registry) {
           registry
               .addInterceptor(interceptor)
               .addPathPatterns("/**")
-              .excludePathPatterns("/actuator/**", "/**/api-docs/**", "/**/swagger-ui/**");
+              .excludePathPatterns(
+                  "/actuator/**", "/**/api-docs/**", "/**/swagger-ui/**", "/favicon.ico", "/error");
```

Comment:
the favicon asset has been added, and thus needs to be provided without auth to enable loading via swagger or (weirdly) /actuator/prometheus also requests the favicon.

Reply:
/error is also added in case any missing route is hit 

/error will be presented to client

but the auth handler interceptor will prevent it from being shown to client as 404 and convert it into a 401 which is confusing. so fixing that here